app/src/main/java/org/isoron/uhabits/models/ScoreList.java

```diff
@@ -89,6 +89,24 @@ public final int getValue(long timestamp)
         return s.getValue();
     }
 
+    /**
+     * Gets the values of all the scores between the given days.
+     *
+     * @param from the timestamp of the first day
+     * @param to the timestamp of the second day
+     * @return the values of all the scores between the given days
+     */
+    public final int[] getValues(long from, long to)
```

Comment:
This method actually returns the values of all the scores that have already been computed. For example, in the following unit test, I am only interested in the values for 3 days:
```
    @Test
    public void test_getValues()
    {
        toggleRepetitions(0, 20);

        long today = DateUtils.getStartOfToday();
        long day = DateUtils.millisecondsInOneDay;

        long from = today - 4 * day;
        long to = today - 2 * day;

        int[] expected = {
            11883254,
            11479288,
            11053198,
        };

        int[] actual = habit.getScores().getValues(from, to);
        assertThat(actual, equalTo(expected));
    }
```
This unit test fails with this error:
```
java.lang.AssertionError: 
Expected: [<11883254>, <11479288>, <11053198>]
     but: was [<12629351>, <12266245>, <11883254>, <11479288>, <11053198>, <10603773>, <10129735>, <9629735>, <9102352>, <8546087>, <7959357>, <7340494>, <6687738>, <5999234>, <5273023>, <4507040>, <3699107>, <2846927>, <1948077>, <1000000>]
```